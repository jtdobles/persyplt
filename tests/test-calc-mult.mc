void main(){
    int a = 3 * 2;
    printi(a);

    float b = 5.0 * 1.5;
    printf(b);
}