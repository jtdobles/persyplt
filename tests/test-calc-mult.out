6
7.500000
